manifests/pipecd/grafana-dashboards/control-plane/overview.json

```diff
@@ -531,7 +531,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "sum by (project) (grpcapi_create_deployment_total)",
+          "expr": "sum by (project) (increase(grpcapi_create_deployment_total[24h]))",
```

Comment:
Should we use `ceil(increase)` instead of just increase, since the value could be not an integer? 🤔

Reply:
I think the `increase` of a counter metric must be an integer, but I'll add the `ceil` operation for safety.